File: SCC0600/exercíciosPy/ex24.py
x = int(input())
maior = 0
menor = 0
soma = 0

for i in range(x):
  n = int(input())
  if i == 0:
    menor = n
    maior = n
  if n > maior:
    maior = n
  if n < menor:
    menor = n
  soma = soma + n

print(maior)
print(menor)
print(soma)

File: SCC0600/exerciciosC/ex24.c
#include <stdio.h>

int main (void) {
    int num; 
    scanf("%d", &num);
    
    int maior = 0;
    scanf("%d", &maior);
    int menor = maior;
    int soma = maior;

    for (int i = 0; i < num - 1; i++) {
        int newnum;
        scanf("%d", &newnum);

        if (newnum > maior) {
            maior = newnum;
        }
        if (newnum < menor) {
            menor = newnum;
        }
        soma += newnum;
    }

    printf("%d\n", maior);
    printf("%d\n", menor);
    printf("%d\n", soma);
}